develop/tutorials/articles/200-integrating-with-front-end-frameworks/03-integrating-with-angular-js.markdown

```diff
@@ -0,0 +1,273 @@
+# Integrating with Angular JS [](id=integrating-with-angular-js)
```

Comment:
The same considerations regarding discouraged `1.x` version of bundler as in the React tutorial apply here.

Also, all references to ES2015+ should be removed because Angular uses Typescript, so there shouldn't be any ES files in the sources.

Reply:
I guess it is because the sample works :-). Why do you think it may be wrong?

Reply:
@izaera I saw that the lib used `es2015` and you said Angular uses TypeScript, so I just wanted to make sure.

Reply:
@mwilliams2014 Mmm... That's an option of the tsc compiler, but has nothing to do with the format of sources/output I believe.

Reply:
Ok great @izaera . I'll wrap these up and push them forward then.